include/freetds/odbc.h

```diff
@@ -568,6 +569,26 @@ TDS_DBC *desc_get_dbc(TDS_DESC *desc);
 /*
  * odbc.c
  */
+/* Temporarily store the bound TVP columns */
+/* Follow the naming in _SQLBindParameter for convenience */
+typedef struct {
+	SQLSMALLINT fParamType; // InputOutputParam
+	SQLSMALLINT fCType;     // C data type
+	SQLSMALLINT fSqlType;   // SQL data type
+	SQLULEN cbColDef;       // columnsize
+	SQLSMALLINT ibScale;    // precision
+	SQLPOINTER rgbValue;    // content buffer
+	SQLLEN cbValueMax;      // buffer length
+	SQLLEN * pcbValue;      // content length buffer
+} SQLTVPCOLUMN;
+
+typedef struct {
+	char * schema;
+	char * typename;
+	int num_cols;
+	SQLTVPCOLUMN ** cols;
+} SQLTVP;
+
```

Comment:
`SQLTVP`/`SQLTVPCOLUMN` structs - stores table and column information in ODBC format